ui/event_handlers.go

```diff
@@ -22,8 +23,8 @@ import (
 )
 
 const maxBarWidth = 50
-const statusSet = common.SpinnerDotSet // SpinnerCircleOutlineSet
-const completedStatus = "✔"            // "●"
+const statusSet = components.SpinnerDotSet // SpinnerCircleOutlineSet
+const completedStatus = "✔"                // "●"
```

Comment:
Just curious: What is `"●"` for?

Reply:
I can remove this --it was used during early development when I was flopping between options